client/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Stage BUILD
+FROM node:18-alpine
```

Comment:
We should also run the client CI/CD with node 18 then, if possible. And don't forget that in the `images.yml` there is a reference to 16.x, change that as well.

Reply:
Exactly, will do that here.